grip/readers.py

```diff
@@ -222,7 +222,9 @@ def is_binary(self, subpath=None):
         Gets whether the specified subpath is a supported binary file.
         """
         mimetype = self.mimetype_for(subpath)
-        return mimetype is not None and mimetype.startswith('image/')
+        if mimetype and mimetype.startswith('text/'):
```

Comment:
```
def is_binary(self, subpath=None):
  return not self.is_text(subpath)

def is_text(self, subpath=None):
  mimetype = self.mimetype_for(subpath) or ''
  return mimetype.startswith('text/')
```

Any idea why this wasn't merged?

Reply:
(aside from passing tests I guess)

Reply:
(Fixed in - thanks for the suggestion, @dmadisetti)